Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/18f_CR_calib_cr_driven_ramsey_RCVersion.py

```diff
@@ -1,12 +1,18 @@
 """
-                                 CR_CALIB_CR_DRIVEN_RAMSEY
+        CR_CALIB_CR_DRIVEN_RAMSEY
 
 The CR_calib scripts are designed for calibrating cross-resonance (CR) gates involving a system
 with a control qubit and a target qubit. These scripts help estimate the parameters of a Hamiltonian,
 which is represented as:
     H = I ⊗ (a_X X + a_Y Y + a_Z Z) + Z ⊗ (b_I I + b_X X + b_Y Y + b_Z Z)
 
-The pulse sequences are as follow:
+The sequence extracts the phase shift due to b_I  Z ⊗ I to enable its correction.
```

Comment:
The sequence extracts the phase shift due to b_I  Z ⊗ I (that arises from light shift) to enable its correction.
no?